packages/elements/src/components/ino-range/ino-range.scss

```diff
@@ -2,23 +2,82 @@
 @use '@material/slider/slider';
 @use '@material/slider' as slider-mixins;
 @use 'base/theme';
+@use "@material/ripple";
 
 @include slider.core-styles;
 
-ino-range {
-  display: block;
-  min-width: 100px;
-
-  @each $color-name in theme.$color-schemes {
-    &.ino-range--color-scheme-#{$color-name} .mdc-slider {
-      @include slider-mixins.track-active-color(theme.color($color-name, base));
-      @include slider-mixins.track-inactive-color(
-        theme.color($color-name, light)
-      );
-      @include slider-mixins.thumb-color(theme.color($color-name, base));
-      @include slider-mixins.thumb-ripple-color(theme.color($color-name, base));
-      @include slider-mixins.tick-mark-inactive-color(#000);
-      @include slider-mixins.value-indicator-text-color(#fff);
+$primary: #2C02FF;
+$primary-focus:#2D5DFF4D;
+$button_filled_hover-control_hover: #4B4EFF;
+$white: #fff;
+$black: #000;
+$transparent: rgba(255, 255, 255, 0);
+$n-8: #F8F9FB;
+$n-7: #EDEFF5;
+$n-6: #CBD0E0;
+$n-5: #ABB1CC;
+$n-4: #8D92B7;
+$n-3: #7275A1;
+$n-2: #565785;
+$n-1: #3A3B64;
```

Comment:
These styles should be extracted to a common stylesheet as they are common in our new designs. Maybe we should start by creating a new theme file (e.g. `base/new-theme.scss`) and using it when updating our components.